addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.xml

```diff
@@ -58,14 +58,12 @@
                         </button>
                     </t>
                     <t t-if="threadViewTopBar.thread and threadViewTopBar.thread.hasInviteFeature">
-                        <Popover popoverClass="'o_ThreadViewTopbar_invitePopover'" position="'bottom'">
-                            <button class="o_ThreadViewTopbar_inviteButton o_ThreadViewTopbar_button" t-att-class="{ 'o-active': threadViewTopBar.threadView.channelInvitationForm.component }" title="Add users" t-on-click="threadViewTopBar.onClickInviteButton">
-                                <i class="fa fa-lg fa-user-plus"/>
-                            </button>
-                            <t t-set="opened">
-                                <ChannelInvitationForm localId="threadViewTopBar.threadView.channelInvitationForm.localId"/>
-                            </t>
-                        </Popover>
+                        <button class="o_ThreadViewTopbar_inviteButton o_ThreadViewTopbar_button" t-att-class="{ 'o-active': threadViewTopBar.isInvitePopoverOpen }" title="Add users" t-on-click="threadViewTopBar.onClickInviteButton" t-ref="inviteButton">
```

Comment:
Ideally use the presence or absence of the popover relation once it is a model. There should be no such thing as a popover being open or not. Either it exists or it doesn't.